setzer/dialogs/document_wizard/pages/page_document_class.py

```diff
@@ -56,20 +56,21 @@ class DocumentClassPageView(PageView):
     def __init__(self):
         PageView.__init__(self)
             
-        self.header.set_text('Choose a document class')
-        self.headerbar_subtitle = 'Step 1: Choose document class'
+        self.header.set_text(_('Choose a document class'))
+        self.headerbar_subtitle = _('Step') + ' 1: ' + _('Choose a document class')
         self.content = Gtk.HBox()
         
         self.list = Gtk.ListBox()
         self.list.set_can_focus(True)
         self.list.set_size_request(348, -1)
         self.list_rows = dict()
-        for name in ['Beamer', 'Letter', 'Book', 'Report', 'Article']:
+        self.document_classes = {_('Beamer'): 'beamer', _('Letter'): 'letter', _('Book'): 'book', _('Report'): 'report', _('Article'): 'article'}
+        for name in self.document_classes:
             label = Gtk.Label(name)
             label.set_xalign(0)
             self.list.prepend(label)
         for row in self.list.get_children():
-            self.list_rows[row.get_child().get_text().lower()] = row
+            self.list_rows[self.document_classes[row.get_child().get_text()]] = row
```

Comment:
This was the only solution I came up with, maybe you have a better idea?

Reply:
I will refactor this later (not in this PR).